.github/CODEOWNERS

```diff
@@ -1 +1,17 @@
 * @nsmithtt
+*.fbs @tapspatel
+/.github/ @vmilosevic
+/include/ttmlir/Conversion/TTNNToEmitC/ @svuckovicTT
+/include/ttmlir/Dialect/TT/ @sdjordjevicTT @rpavlovicTT @mrakitaTT
+/include/ttmlir/Dialect/TTIR/ @sdjordjevicTT @rpavlovicTT @mrakitaTT
```

Comment:
All 3 of you are listed as "Dialect" owners.  The `TT` dialect might be OK to have many eyes on since it's kind of used across the board, but maybe we should pick 1 person for `TTIR` and 1 person for `TTNN`.

@mrakitaTT, I have you temporarily listed here for now, but I imagine once we have stableHLO conversion in place you'll be the exclusive owner there.